### PR TITLE
adding omsagnet image ciprod08052021-1

### PR DESCRIPTION
**Pull request by daweim0**
Title says it all. The only difference between ciprod08052021 and ciprod08052021-1 is the image tag

@ganga1980 @vishiy @rashmichandrashekar @gracewehner

### CODE REVIEW

**Review by ganga1980 — APPROVED** [review]

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -144,7 +144,8 @@
         "ciprod06112021",
         "ciprod05202021-hotfix",
         "ciprod06112021-1",
-        "ciprod08052021"
+        "ciprod08052021",
```
Comment (ganeshkumar5699) [inline vhdbuilder/packer/components.json:147]: Is it ok to have this older version? And can that be removed since it will not be used?

**Review by anujmaheshwari1 — APPROVED** [review]
lgtm pending all VHDs build successfully!

**Review by chengliangli0918 — APPROVED** [review]

File: vhdbuilder/packer/components.json (hunk 1 above)
Comment (ganga1980) [reply vhdbuilder/packer/components.json:147]: yes, ciprod08052021 ok to have and we will remove old ones next agent release.

Comment (daweim0) [reply vhdbuilder/packer/components.json:147]: The older version will still be used. We only plan to use the newer version if there is an issue toggling a few customers to the new version

**Review by ganeshkumar5699 — APPROVED** [review]
Lgtm. And thanks for the info about the previous version and the plan to remove the older versions.

**Review by ganga1980 — APPROVED** [review]

**Review by juan-lee — APPROVED** [review]

**Pull request merged**